src/coroutine.js

```diff
@@ -1,25 +1,21 @@
+import Action from './Action'
+
 export default function (resolve, iterator, promise) {
 	new Coroutine(resolve, iterator, promise).run()
+	// taskQueue.add(new Coroutine(resolve, iterator, promise))
```

Comment:
I've considered this as well.  I'd be interested to hear your thoughts on always queueing coroutines vs not.